src/components/Calendar.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+
+import FullCalendar from 'fullcalendar-reactwrapper';
+import 'fullcalendar-reactwrapper/dist/css/fullcalendar.min.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+import axios from 'axios';
+
+import Tooltip from './Tooltip.jsx';
+
+const CALENDAR_ID = '8n8u58ssric1hmm84jvkvl9d68@group.calendar.google.com';
```

Comment:
Definitely shouldn't be committing these

Reply:
Moved to .env file